etc/init.d/contextBroker.centos

```diff
@@ -141,8 +141,7 @@ broker_coredump_start()
         ulimit -c unlimited
         ulimit -n 10240
         ${EXECUTABLE} ${BROKER_OPTS} -fg &> /dev/null &
-        sleep 2 # wait some time to let the contextbroker start
-        ps -ef | grep ${EXECUTABLE} | grep -v grep | awk '{print $2}' > ${BROKER_PID_FILE}
+        sleep 2 # wait some time to let the contextbroker start        
```

Comment:
Why not do a smarter sleep?
A while with a timeout checking the broker port using nc, as we do in other places?